server/src/test/java/org/opensearch/bootstrap/IdentityPluginTests.java

```diff
@@ -21,26 +22,29 @@
 
 import static org.hamcrest.Matchers.instanceOf;
 import static org.hamcrest.Matchers.is;
+import static org.mockito.Mockito.mock;
 
 public class IdentityPluginTests extends OpenSearchTestCase {
 
     public void testSingleIdentityPluginSucceeds() {
+        Client client = mock(Client.class);
         TestThreadPool threadPool = new TestThreadPool(getTestName());
```

Comment:
I don't think this is used any more, other than terminating it.  (Also on line 42.)

Should the mocked client return this thread pool to ensure the next lines are actually testing against a real thread pool?

Reply:
Added `when(client.threadPool()).thenReturn(threadPool);`